packages/js/src/ui/index.css

```diff
@@ -0,0 +1,469 @@
+/*
+! tailwindcss v3.4.4 | MIT License | https://tailwindcss.com
+*/ /*
+1. Prevent padding and border from affecting element width. (https://github.com/mozdevs/cssremedy/issues/4)
+2. Allow adding a border to an element by just adding a border-width. (https://github.com/tailwindcss/tailwindcss/pull/116)
+*/
+
+.novu {
```

Comment:
❓ **question:** Where did we get this CSS reset code from?

Reply:
This is just the default tailwind reset, but we're using the where selector to get specificity down to 0 to allow overriding.